registration-processor/pom.xml

```diff
@@ -132,18 +132,18 @@
 		<kernel.templatemanager.velocity.version>1.3.0-beta.1</kernel.templatemanager.velocity.version>
 		<kernel.idvalidator.uin.version>1.3.0-beta.1</kernel.idvalidator.uin.version>
 		<kernel.idvalidator.rid.version>1.3.0-beta.1</kernel.idvalidator.rid.version>
-		<kernel.cbeffutil.version>1.3.0-SNAPSHOT</kernel.cbeffutil.version>
+		<kernel.cbeffutil.version>1.3.0-beta.1</kernel.cbeffutil.version>
 		<kernel.bioapi.provider.version>1.3.0-beta.1</kernel.bioapi.provider.version>
-		<kernel.auth.adapter.version>1.3.0-SNAPSHOT</kernel.auth.adapter.version>
+		<kernel.auth.adapter.version>1.3.0-beta.1</kernel.auth.adapter.version>
 		<kernel.pdfgenerator.itext.version>1.3.0-beta.1</kernel.pdfgenerator.itext.version>
 		<kernel.qrcodegenerator.zxing.version>1.3.0-beta.1</kernel.qrcodegenerator.zxing.version>
 		<kernel.logger.logback.version>1.3.0-beta.1</kernel.logger.logback.version>
 		<kernel.idvalidator.vid.version>1.3.0-beta.1</kernel.idvalidator.vid.version>
-		<object.store.khazana.version>1.3.0-SNAPSHOT</object.store.khazana.version>
+		<object.store.khazana.version>1.3.0-beta.1</object.store.khazana.version>
 		<kernel.biometrics.api.version>1.3.0-SNAPSHOT</kernel.biometrics.api.version>

```

Comment:
Looks like <kernel.biometrics.api.version> is also frombio-utils. Can we change it to 1.3.0-beta.1.

Reply:
done